recipes/ckanweb-deploy.rb

```diff
@@ -100,7 +100,7 @@
 # Create Apache config files
 #
 
-template "#{config_dir}/apache.wsgi" do
+template "#{config_dir}/wsgi.py" do
```

Comment:
so py3 only on this?

Reply:
No, it will work fine with any Python version. I adjusted the name so it would match the naming convention [introduced by CKAN 2.9](https://github.com/ckan/ckan/blob/master/wsgi.py).